tests/unit/services/userService.test.ts

```diff
@@ -203,7 +203,6 @@ describe('UserService', () => {
 		test('Fails to create user profile with invalid details', async () => {
 			await expect(userService.putUserProfile(userWithoutProfile.id, {} as any)).rejects.toMatchObject({ httpCode: HttpCode.BadRequest });
 			await expect(userService.putUserProfile(userWithoutProfile.id, { course: 'History' } as any)).rejects.toMatchObject({ httpCode: HttpCode.BadRequest });
```

Comment:
Ahh, to get this test to work can you also add a yearOfStudy property? I just remembered that this will fail if course and yearOfStudy are not passed. To make sure that it's failing only because of `course`, could you add in a valid yearOfStudy?